samples/backend/python/quart/__init__.py

```diff
@@ -0,0 +1,181 @@
+import base64
+import json
+import os
+import re
+from dotenv import load_dotenv
```

Comment:
If you want to adhere to Python sample guidelines, use ruff/black/isort to style and lint the code. This file has unsorted imports, for examples.

Guidelines here:
https://github.com/Azure-Samples/samples-guidelines/blob/main/python-guidelines.md

Example config @ https://github.com/pamelafox/python-project-template

Reply:
will do, thanks!